lnd.go

```diff
@@ -259,18 +260,6 @@ func Main(cfg *Config, lisCfg ListenerCfg, interceptor signal.Interceptor) error
 	if err := cfg.DB.Init(ctx, cfg.localDatabaseDir()); err != nil {
 		return err
 	}
-
-	localChanDB, remoteChanDB, cleanUp, err := initializeDatabases(ctx, cfg)
-	switch {
-	case err == channeldb.ErrDryRunMigrationOK:
-		ltndLog.Infof("%v, exiting", err)
-		return nil
-	case err != nil:
-		return fmt.Errorf("unable to open databases: %v", err)
-	}
-
-	defer cleanUp()
-
 	// Only process macaroons if --no-macaroons isn't set.
```

Comment:
nit: leave one blank line before the comment.

Reply:
done